lib/samples/style_graphics_with_symbols/style_graphics_with_symbols.dart

```diff
@@ -0,0 +1,322 @@
+    final redCircleSymbol = SimpleMarkerSymbol(
+      style: SimpleMarkerSymbolStyle.circle,
+      color: Colors.red,
+      size: 10,
+    );
+
+    // Create graphics and add them to graphics overlay.
+    var graphic = Graphic(
+      geometry: ArcGISPoint(
+        x: -2.72,
+        y: 56.065,
+        spatialReference: SpatialReference.wgs84,
+      ),
+      symbol: redCircleSymbol,
+    );
+    _graphicsOverlay.graphics.add(graphic);
+
+    graphic = Graphic(
+      geometry: ArcGISPoint(
+        x: -2.69,
+        y: 56.065,
+        spatialReference: SpatialReference.wgs84,
+      ),
+      symbol: redCircleSymbol,
+    );
+    _graphicsOverlay.graphics.add(graphic);
+
+    graphic = Graphic(
+      geometry: ArcGISPoint(
+        x: -2.66,
+        y: 56.065,
+        spatialReference: SpatialReference.wgs84,
+      ),
+      symbol: redCircleSymbol,
+    );
+    _graphicsOverlay.graphics.add(graphic);
+
+    graphic = Graphic(
+      geometry: ArcGISPoint(
+        x: -2.63,
+        y: 56.065,
+        spatialReference: SpatialReference.wgs84,
+      ),
+      symbol: redCircleSymbol,
+    );
+    _graphicsOverlay.graphics.add(graphic);
+  }
+
+  void _createPolyline() {
+    // Create a purple simple line symbol.
+    final lineSymbol = SimpleLineSymbol(
+      style: SimpleLineSymbolStyle.dash,
+      color: Colors.purple,
+      width: 10,
+    );
+
+    // Build a polyline.
+    final polylineBuilder =
+        PolylineBuilder(spatialReference: SpatialReference.wgs84);
+    polylineBuilder.addPointXY(
+      x: -2.715,
+      y: 56.061,
+    );
+    polylineBuilder.addPointXY(
+      x: -2.6438,
+      y: 56.079,
+    );
+    polylineBuilder.addPointXY(
+      x: -2.638,
+      y: 56.079,
+    );
+    polylineBuilder.addPointXY(
+      x: -2.636,
+      y: 56.078,
+    );
+    polylineBuilder.addPointXY(
+      x: -2.636,
+      y: 56.077,
+    );
+    polylineBuilder.addPointXY(
+      x: -2.637,
+      y: 56.076,
+    );
+    polylineBuilder.addPointXY(
+      x: -2.715,
+      y: 56.061,
+    );
+
+    final polyline = polylineBuilder.toGeometry();
+
+    // Create the graphic with polyline and symbol.
+    final graphic = Graphic(
+      geometry: polyline,
+      symbol: lineSymbol,
+    );
+
+    // Add graphic to the graphics overlay.
+    _graphicsOverlay.graphics.add(graphic);
+  }
+
+  void _createPolygon() {
+    // Create a green dash line symbol
+    final outlineSymbol = SimpleLineSymbol(
+      style: SimpleLineSymbolStyle.dash,
+      color: Colors.green,
+      width: 1,
+    );
+
+    // Create a green mesh simple fill symbol.
+    final fillSymbol = SimpleFillSymbol(
+      style: SimpleFillSymbolStyle.diagonalCross,
+      color: Colors.green,
+      outline: outlineSymbol,
+    );
+
+    // Create a new point collection for polygon.
+    final polygonBuilder =
+        PolygonBuilder(spatialReference: SpatialReference.wgs84);
+    polygonBuilder.addPointXY(
+      x: -2.6425,
+      y: 56.0784,
+    );
```

Comment:
This can all fit on one line by removing the `,` at the end and then letting it get reformatted.
```dart
polygonBuilder.addPointXY(x: -2.6425, y: 56.0784); 
```
That's how we've done these bulk data points in other samples